app/src/main/java/org/mozilla/fenix/share/ShareInteractor.kt

```diff
@@ -12,7 +12,7 @@ import org.mozilla.fenix.share.listadapters.AppShareOption
  */
 class ShareInteractor(
     private val controller: ShareController,
-) : ShareCloseInteractor, ShareToAccountDevicesInteractor, ShareToAppsInteractor {
+) : ShareCloseInteractor, ShareToAccountDevicesInteractor, ShareToAppsInteractor, ShareSaveToPDFInteractor {
```

Comment:
These class naming conventions do not make sense since we're already in the `share` package. 😄